cmd/flags.go

```diff
@@ -54,6 +54,7 @@ func initGlobalFlags(globalFlags *pflag.FlagSet) {
 	globalFlags.Bool("no-data", false, "Open app without data")
 	globalFlags.Bool("bash", false, "Bash flag used to adapt output to bash completion format")
 	globalFlags.MarkHidden("bash")
+	globalFlags.String("context", "", "Specific context that should be used when connecting")
```

Comment:
=> "Name of context to use when connecting to the Qlik Associative Engine" or something similar?

Reply:
Updated to more descriptive text similar to that of engine url.